custom/enikshay/exceptions.py

```diff
@@ -12,3 +12,7 @@ class NikshayLocationNotFound(ENikshayException):
 
 class NikshayCodeNotFound(ENikshayException):
     pass
+
+
+class RequiredValueMissing(ENikshayException):
```

Comment:
nit: Might be clearer if this was called `NikshayRequiredValueMissing`, so that it fits the pattern of the rest of the exceptions